__tests__/screens/FeedScreen/HeaderFeed.test.js

```diff
@@ -9,12 +9,14 @@ jest.mock('react-native-activity-feed/node_modules/react-native-image-crop-picke
   openPicker: () => jest.fn()
 }));
 const mockedGoBack = jest.fn();
+const mockedNavigate = jest.fn();
 
 jest.mock('@react-navigation/native', () => ({
   ...jest.requireActual('@react-navigation/native'),
   useRoute: () => jest.fn(),
   useNavigation: () => ({
-    goBack: mockedGoBack
+    goBack: mockedGoBack,
+    navigate: mockedNavigate
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `navigate` function is now being mocked in the `useNavigation` hook. This change allows for testing navigation actions within the component, which can be useful for ensuring that the correct screens are navigated to under certain conditions.
```diff
+ const mockedNavigate = jest.fn();
  
  jest.mock('@react-navigation/native', () => ({
    ...jest.requireActual('@react-navigation/native'),
    useRoute: () => jest.fn(),
    useNavigation: () => ({
      goBack: mockedGoBack,
+     navigate: mockedNavigate
    })
  }));
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->